client/lib/importer/actions.js

```diff
@@ -56,7 +57,7 @@ export function cancelImport( siteId, importerId ) {
 		.catch( apiFailure );
 }
 
-export function failUpload( importerId, error ) {
+export function failUpload( importerId, { message: error } ) {
```

Comment:
do you think it would be better if I got rid of the renaming in the argument? I mainly did that instead of doing it on the other side…

```js
failUpload( importerId, error ) {
	…
	{ error: message }
}
```